app/store/index.js

```diff
@@ -1,9 +1,15 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import PapaParse from "papaparse";
+import axios from 'axios'
+import { vuexfireMutations, firestoreAction } from 'vuexfire'
+import db from "../src/plugins/database"
 
-Vue.use(Vuex)
+
+Vue.use(Vuex, axios, vuexfireMutations)
```

Comment:
axios

Reply:
we still need to keep axios in there to send https request. That is seperate from the web sockets